google/cloud/bigtable/table.cc

```diff
@@ -13,11 +13,13 @@
 // limitations under the License.
 
 #include "google/cloud/bigtable/table.h"
+
```

Comment:
I thought we weren't doing this (yet)?

Reply:
Somehow I picked up a clang-format from the future. Fixed.